CondFormats/EcalObjects/interface/EcalChannelStatusCode.h

```diff
@@ -5,6 +5,7 @@
  * Created: 14 Nov 2006
```

Comment:
looks like the only changes to this file are in whitespaces; could you revert them ?

Reply:
Done and it will come in the next commit.